packages/components/src/server-side-render/index.js

```diff
@@ -34,6 +34,7 @@ export class ServerSideRender extends Component {
 		this.state = {
 			response: null,
 		};
+		this.fetch = debounce( this.fetch, 500 );
```

Comment:
The assignment of `this.isStillMounted` in `componentWillUnmount` is not enough for this, since there are logic paths of `fetch` which call `this.setState` directly without consideration of the instance value, thus resulting in errors / warnings.

We should do one of:

- Always consider `this.isStillMounted` at the beginning of `fetch` and in its callback handlers.
- Call `this.fetch.cancel()` in `componentWillUnmount`

Reply:
The `setState` calls have a check for `this.isStillMounted` in the callbacks already, https://github.com/WordPress/gutenberg/blob/1084fe302b83affb547d22f193591eb1e2fa258a/packages/components/src/server-side-render/index.js#L65

Is there something I've missed there?